package.json

```diff
@@ -25,7 +25,7 @@
     "dist"
   ],
   "scripts": {
-    "start": "cross-env TARGET=es rollup -c -w",
+    "start": "concurrently \"tsc --emitDeclarationOnly -w\" \"cross-env TARGET=es rollup -c -w\"",
```

Comment:
Ignore this. It's another commit that I intended to push into master ca4350b4a6ef66ad994d1749149f6c8396302711. Github's diff is wired sometimes.